server/sendEmail.js

```diff
@@ -0,0 +1,13 @@
+require("dotenv").config();
+const sgMail=require('@sendgrid/mail');
+sgMail.setApiKey('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');
+const message={
+    to:'gj7097@srmist.edu.in',
```

Comment:
Isn't the "to" supposed to be the email of the user?

Reply:
Actually the sendEmail.js file is a test file to see if the basic sending of email is working, thats why "to" has been hardcoded

Reply:
I see...can you rename the file to sendEmail.test.js, in order to avoid confusion? 
and make sure that whatever test files you have, are appended with ".test" in their file name